Dockerfile

```diff
@@ -20,15 +20,19 @@ ENV PHANTOMJS_VERSION 2.1.1
 RUN curl -sL https://deb.nodesource.com/setup_8.x | bash -
 RUN apt-get update -qq && apt-get install -y build-essential bundler libmariadbclient-dev ruby-rmagick libfreeimage3 wget curl procps cron make nodejs strace
 RUN wget https://github.com/Medium/phantomjs/releases/download/v$PHANTOMJS_VERSION/phantomjs-$PHANTOMJS_VERSION-linux-x86_64.tar.bz2 -O /tmp/phantomjs-$PHANTOMJS_VERSION-linux-x86_64.tar.bz2; tar -xvf /tmp/phantomjs-$PHANTOMJS_VERSION-linux-x86_64.tar.bz2 -C /opt ; cp /opt/phantomjs-$PHANTOMJS_VERSION-linux-x86_64/bin/* /usr/local/bin/
-RUN npm install -g bower
+
+# Install yarn
+RUN npm config set strict-ssl false
+RUN npm install -g yarn
 
 # Install bundle of gems
 WORKDIR /tmp
 ADD Gemfile /tmp/Gemfile
-ADD Gemfile.lock /tmp/Gemfile.lock
-RUN bundle install --jobs 4
+#ADD Gemfile.lock /tmp/Gemfile.lock  # *DISABLED TO TEST IMAGE SWITCH* 
```

Comment:
This is likely the problem! `mini_mime` is only declared in `Gemfile.lock`!
Hope it helps!

Reply:
Ok lets see if it works by removing this comment